### DISCUSSION TITLE
Integrating Apache Echarts in Superset

### DISCUSSION THREAD

**Discussion by mayanklalwani** [apache/superset#29259] [Q&A / Help]
I have a superset app running, I am looking for an option to integrate charts from apache echarts into my superset.

Can someone please help me if there are steps listed down anywhere to do this or some documentation available?

**Comment by shawnngtq** [comment]
@mayanklalwani ,

Did you find any guides / steps?

**Reply by mayanklalwani** [reply]
@shawnngtq - No, couldn't find any guide for this.

**Comment by Nakulp07** [comment]
@mayanklalwani I'm also looking for same functionality.
Did you find anything?

**Comment by sfirke** [comment]
Can you say more about what you want to accomplish?  For several years, Superset has been migrating its charts to use ECharts.  For instance: https://preset.io/blog/2021-6-14-superset-nine-new-charts/
